site/client/owid.entry.ts

```diff
@@ -53,9 +53,21 @@ window.runRelatedCharts = runRelatedCharts
 window.runLightbox = runLightbox
 window.runCovid = runCovid
 window.runGlobalEntityControl = runGlobalEntityControl
+window.runPageTooltips = runPageTooltips
 
 Analytics.logPageLoad()
 
+document.querySelector("html")?.classList.add("js")
```

Comment:
Interesting. Any idea what this legacy code is doing?

Reply:
It allows you to style the page differently in CSS based on whether JavaScript is enabled / has loaded.

It _seems_ the CSS classes `.js` and `.no-js` are not used anywhere, but maybe let's make that change outside of this PR?

Reply:
For a little bit more context on this one @breck7:
There is the `noscript` tag which allows fallback rendering in case the browser has JavaScript disabled entirely.
But there's also the case where JS is theoretically enabled, but the script cannot be processed without errors (IE11!). We cannot detect this with a pure `noscript` tag, but we can with this extra little code bit.
Whether we use it or not is a different story :)